### ISSUE TITLE
Recommend asking about name pronunciation

### ISSUE THREAD

**Issue by aartikelapure** [Bloom-Works/handbook#140]
The final bullet point in the section [What we expect and why](https://bloom-handbook.netlify.app/sections/code-of-conduct/#what-we-expect-and-why) says:

- Treat others’ identities and cultures with respect. Make an effort to say people's names correctly and refer to them by their stated pronouns.

I'd like to suggest editing "Make an effort to say people's names correctly" to something like "If you are unsure how to pronounce someone's name, ask them how they would like their name pronounced and make an effort to say it correctly." 

Basically, don’t assume you know what “correct” pronunciation is. 

I have personal feelings about this one from several bad workplace experiences from well-intentioned colleagues who decided what correct pronunciation was for me. I am happy to share if more clarification or explanation is needed to make an edit.

**Comment by dottiebobottie** [comment]
Updated in https://github.com/bloom-works/handbook/blob/pumpkin-patches/sections/code-of-conduct.md

**Comment by danielhonker** [comment]
In #147
